python/doc/theory/data_analysis/smirnov_test.rst

```diff
@@ -1,52 +1,63 @@
 .. _smirnov_test:
 
-Kolmogorov-Smirnov two samples test
------------------------------------
-
-Let :math:`X` be a scalar uncertain variable modeled as a random
-variable. This method deals with the construction of a dataset prior to
-the choice of a probability distribution for :math:`X`. This statatistical
-test is used to compare two samples :math:`\left\{x_1,\ldots,x_N \right\}`
-and :math:`\left\{x'_1,\ldots,x'_M \right\}`; the goal is to determine
-whether these two samples come from the same probability distribution or
-not. If this is the case, the two samples should be aggregated in order
-to increase the robustness of further statistical analysis.
-
-The test relies on the maximum distance between the cumulative distribution
-functions :math:`\widehat{F}_N` and :math:`\widehat{F}'_M` of the samples
-:math:`\left\{x_1,\ldots,x_N \right\}` and :math:`\left\{x'_1,\ldots,x'_M \right\}`.
-This distance is expressed as follows:
+Kolmogorov-Smirnov test
```

Comment:
This cannot be the title of this document, because there would be two different help pages with the same title: please look at kolmogorov_test.rst. The previous title was more accurate in my opinion, but it can be more accurate (see [Wikipedia's page](https://en.wikipedia.org/wiki/Kolmogorov%E2%80%93Smirnov_test#Two-sample_Kolmogorov%E2%80%93Smirnov_test)).

```suggestion
Two-sample Kolmogorov-Smirnov test
```

Please do not forget to adapt the underline string below to adapt to the new length.

Reply:
ok